src/main/java/ingest/utility/IngestUtilities.java

```diff
@@ -120,6 +120,8 @@
 	private String AMAZONS3_BUCKET_NAME;
 	@Value("${vcap.services.pz-blobstore.credentials.encryption_key}")
```

Comment:
I noticed in the other PR we made changes here, but it looks like they weren't necessary here, correct?